src/components/Bullet.tsx

```diff
@@ -442,11 +441,78 @@ const Bullet = ({
     return children.length < Object.keys(thought.childrenMap).length
   })
 
-  // fill =bullet/=style override
+  /** Check if the entire value is wrapped in a single font or span tag. */
+  const isWrappedInSingleTag = (str: string): boolean => {
```

Comment:
This seems so provide similar functionality as https://github.com/cybersemics/em/blob/main/src/util/getCommandState.ts. Is there a reason that can't be used?

Reply:
Literally, `isWrappedInSingleTag` checks if the thought value is wrapped in a sing tag.
Like this : <font><span>He</span><font>ll</font>o</span>   : true
               <font>Hel</font><span>lo</span>  : false
And `getCommandState` function determines which commands (bold, italic, underline ....) applied to the thought value.
So, I can't use `getCommandState` in `Bullet.tsx`

Reply:
Are you saying that `getCommandState` also returns the color of substrings, not just the whole string? I think we're saying the same thing, just in different ways, but correct me if I'm wrong.

Reply:
Yes, it also returns the color of substrings.
So, I can't use `getCommandState` in Bullet.